app/server/controllers/login.ts

```diff
@@ -123,7 +123,10 @@ function getIdamCallback(
           body.filter(hearing => hearing.case_id + '' === req.query.caseId) : body;
 
         hearings.forEach(value => {
-          value.case_reference = (value.case_reference) ? value.case_reference : value.case_id.toString();
+          value.case_reference = value.case_id ? value.case_id.toString() : '';
+          if (value.case_reference === '') {
```

Comment:
Why we need to throw error here? was it a new requirement

Reply:
No, will check with Hemingway or Rosie